rally-domain/src/main/groovy/yakworks/rally/orgs/model/Company.groovy

```diff
@@ -28,8 +28,9 @@ class Company implements NameNum, RepoEntity<Company>, Serializable {
     }
 
     static constraintsMap = [
-        website:[ description: 'The company website', nullable: true],
-        sourceId:[ description: 'The source identifier for this company', nullable: true]
+        name:[ d: 'Full name for this company', maxSize: 255],
```

Comment:
can you check, does this fully override or does it merge? meaning, can we just specify the description and let the maxSize come from the trait? I cant recall if we merge or replace the map.

Reply:
@basejump i see, it merges, so values from traits are merged, and class level overrides the traits

Did some cleanup
- removed maxSize where it was same as trait
- removed constraint from class where size, description all was same as trait